### PR TITLE
Add docs page for Get Top Coin Holders

### PR DESCRIPTION
**Pull request by jossifelefteriadis**

### CODE REVIEW

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+<SetupMoralis node="moralis" python="moralis" />
+
+## Step 2: Get Top Coin Holders
+
+In order to get top coin holders, Moralis provides you a [getTopHoldersByCoin](/web3-data-api/aptos/reference/get-top-holders-by-coin) endpoint to do so.
+
+Here you'll need two parameters: `limit` and `coinTypeHash`.
+
+Once you have obtained both the `limit` and `coinTypeHash`, you can copy the following code:
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:27]:
```suggestion
Here you'll need two parameters: `limit`, `coinTypeHash`, and `network`.

Once you have obtained both the `limit`, `coinTypeHash`, and `network`, you can copy the following code:
```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  await Moralis.start({
+    apiKey: "YOUR_API_KEY",
+    // ...and any other configuration
+  });
+
+  const limit = 10;
+
+  const coinTypeHash = 
+    "91ceb1308a98389691e05158b07ed5f079ab78461a6bb8d5a4054b1bb5cb8bb6",
+  ;
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:48]:
```suggestion
  const coinTypeHash = "91ceb1308a98389691e05158b07ed5f079ab78461a6bb8d5a4054b1bb5cb8bb6";
```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+  const response = await Moralis.AptosApi.coins.getTopHoldersByCoin({
+    limit,
+    coinTypeHash,
+    network: "mainnet"
+  });
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:54]:
```suggestion
  const network = "mainnet";

  const response = await Moralis.AptosApi.coins.getTopHoldersByCoin({
    limit,
    coinTypeHash,
    network,
  });
```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+  await Moralis.start({
+    apiKey: "YOUR_API_KEY",
+    // ...and any other configuration
+  });
+
+  const limit = 10;
+
+  const coinTypeHash = 
+    "91ceb1308a98389691e05158b07ed5f079ab78461a6bb8d5a4054b1bb5cb8bb6",
+  ;
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:78]:
```suggestion
  const coinTypeHash = "91ceb1308a98389691e05158b07ed5f079ab78461a6bb8d5a4054b1bb5cb8bb6";
```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+  const response = await Moralis.AptosApi.coins.getTopHoldersByCoin({
+    limit,
+    coinTypeHash,
+    network: "mainnet"
+  });
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:84]:
```suggestion
  const network = "mainnet";
  const response = await Moralis.AptosApi.coins.getTopHoldersByCoin({
    limit,
    coinTypeHash,
    network,
  });
```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+import SetupMoralis from '/docs/partials/\_install-moralis-sdk.mdx';
+
+<SetupMoralis node="moralis" python="moralis" />
+
+## Step 2: Get Top Coin Holders
+
+In order to get top coin holders, Moralis provides you a [getTopHoldersByCoin](/web3-data-api/aptos/reference/get-top-holders-by-coin) endpoint to do so.
+
+Here you'll need two parameters: `limit`, `coinTypeHash`, and `network`.
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:25]:
```suggestion
Here you'll need three parameters: `limit`, `coinTypeHash`, and `network`.
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+## Step 1: Setup Moralis
+
+import SetupMoralis from '/docs/partials/\_install-moralis-sdk.mdx';
+
+<SetupMoralis node="moralis" python="moralis" />
+
+## Step 2: Get estimate gas price
+
+In order to get estimate gas price, Moralis provides you a [estimateGasPrice](/web3-data-api/aptos/reference/estimate-gas-price) endpoint to do so.
+
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:24]:
```suggestion

Here you'll need one parameter: `network`.

Once you have obtained the `network`, you can copy the following code:

```

File: docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+Once you have obtained both the `limit`, `coinTypeHash`, and `network`, you can copy the following code:
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/coin-api/04-Get Coin Owners/how-to-get-top-coin-holders.md:27]:
```suggestion
Once you have obtained the `limit`, `coinTypeHash`, and `network`, you can copy the following code:
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+const runApp = async () => {
+  await Moralis.start({
+    apiKey: "YOUR_API_KEY",
+    // ...and any other configuration
+  });
+
+  const response = await Moralis.AptosApi.transactions.estimateGasPrice({
+    "network": "mainnet"
+  });
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:42]:
```suggestion
  const network = "mainnet";

  const response = await Moralis.AptosApi.transactions.estimateGasPrice({
    network,
  });
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
+---
+title: "How to get estimate gas price"
+slug: "../how-to-get-estimate-gas-price"
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:3]:
```suggestion
title: "How to estimate the gas price"
slug: "../how-to-estimate-the-gas-price"
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+description: "Learn how to get estimate gas price using Moralis Transaction API."
+sidebar_label: Get Estimate Gas Price
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:5]:
```suggestion
sidebar_label: Estimate Gas Price
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+- Node v.14+ or Python
+- NPM/Yarn or Pip
+
+## Step 1: Setup Moralis
+
+import SetupMoralis from '/docs/partials/\_install-moralis-sdk.mdx';
+
+<SetupMoralis node="moralis" python="moralis" />
+
+## Step 2: Get estimate gas price
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:21]:
```suggestion
## Step 2: Estimate gas price
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+In order to get estimate gas price, Moralis provides you a [estimateGasPrice](/web3-data-api/aptos/reference/estimate-gas-price) endpoint to do so.
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:23]:
```suggestion
In order to estimate the gas price, Moralis provides you a [estimateGasPrice](/web3-data-api/aptos/reference/estimate-gas-price) endpoint to do so.
```

File: docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+    "x-aptos-ledger-oldest-version": "0",
+    "x-aptos-ledger-timestampusec": "1681312155079381",
+    "x-aptos-epoch": "2186",
+    "x-aptos-block-height": "46554718",
+    "x-aptos-oldest-block-height": "0"
+  }
+}
+```
+
+Congratulations 🥳 You just got estimate gas price with just a few lines of code using the Moralis Transaction API!
```
Comment (YosephKS) [inline docs/01-web3-data-api/aptos/transactions-api/how-to-get-estimate-gas-price.md:124]:
```suggestion
Congratulations 🥳 You just estimate the gas price with just a few lines of code using the Moralis Transaction API!
```

**Review by YosephKS — CHANGES_REQUESTED** [review]
Just some minor fix here and there @jossifelefteriadis

**Comment by jossifelefteriadis** [comment]
> Just some minor fix here and there @jossifelefteriadis

Fixed @YosephKS

**Review by YosephKS — APPROVED** [review]

**Pull request merged**